content/docs/next-release-v2/storage.md

```diff
@@ -26,9 +26,14 @@ For large scale production deployment the Jaeger team [recommends OpenSearch bac
 
 ## Remote Storage
 
-Jaeger supports a gRPC-based [Remote Storage API][storage.proto] that allows extending the Jaeger ecosystem with custom storage backends, not directly supported by the project. These storage backends can be deployed as a remote gRPC server.
-
-To use a remote storage as Jaeger storage backend, use `grpc` as the storage type and specify the remote gRPC server address. For more information, please refer to [jaeger/internal/storage/v1/grpc](https://github.com/jaegertracing/jaeger/tree/main/internal/storage/v1/grpc).
+Jaeger supports a gRPC-based Remote Storage API that allows extending the Jaeger
```

Comment:
```suggestion
Jaeger supports a gRPC-based Remote Storage API v2 that allows extending the Jaeger
```

Reply:
and original text had a link to the proto: `[Remote Storage API][storage.proto]`

Reply:
@yurishkuro I removed that because we have more than one proto file now which are listed in more detail in the README. Do you want to copy some of that detail over here?

Reply:
ok

Reply:
I would include the proto links, because this page describes the API, it's worth having fewer clicks.

Reply:
Done